client/i18n/locales/english/intro.json

```diff
@@ -66,44 +86,198 @@
       },
       "learn-html-by-building-a-cat-photo-app": {
```

Comment:
I understand the duplicate certification project intros, but seeing as these "do not" (should not) appear within `responsive-web-design`, can these not be removed?